packages/compass-crud/src/components/crud-toolbar.tsx

```diff
@@ -173,8 +173,10 @@ const CrudToolbar: React.FunctionComponent<CrudToolbarProps> = ({
   const nextButtonDisabled = useMemo(
     // If we don't know the count, we can't know if there are more pages.
     () =>
-      count === undefined || count === null ? false : 20 * (page + 1) >= count,
```

Comment:
OMG now I'm trying to fight the urge to search the codebase for magic numbers and audit all of them..